healdata.org/metadata/aggregate_config.json

```diff
@@ -7,14 +7,14 @@
 		},
 		"JCOIN": {
 			"mds_url": "https://jcoin.datacommons.io/",
-			"commons_url" : "jcoin.datacommons.io",
+            "commons_url": "jcoin.datacommons.io",
```

Comment:
taking off the white space before `:` makes sense to me, but the change of preceeding whitespaces doesn't

if this file in preprod's manifest is ill-formated, we should fix from there, not introducing these format into anoter file

Reply:
will revert this ... tbh I ran a job to get all the differences and these 2 were the only ones that seemed valid, did not really check the details :)